x-pack/elastic-agent/pkg/agent/program/testdata/endpoint_no_fleet.yml

```diff
@@ -0,0 +1,88 @@
+name: Endpoint Host
+
+outputs:
+  default:
+    type: elasticsearch
+    hosts: [127.0.0.1:9200, 127.0.0.1:9300]
+    username: elastic
+    password: changeme
+    api_key: TiNAGG4BaaMdaH1tRfuU:KnR6yE41RrSowb0kQ0HWoA
+    ca_sha256: 7HIpactkIAq2Y49orFOOQKurWxmmSFZhBCoQYcRhJ3Y=
+
+datasources:
```

Comment:
Probably this needs an update after the PR merged by @michalpristas recently.

Reply:
Done